src/Tizen/Avalonia.Tizen/Platform/ScalingInfo.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Tizen.System;
+using ElmSharp;
+
+namespace Avalonia.Tizen.Platform;
+public static class ScalingInfo
```

Comment:
I think it's not supposed to be public. Other classes should be also double-checked if visibility is expected.

Reply:
File removed.